chord_metadata_service/chord/api_views.py

```diff
@@ -109,3 +109,15 @@ def destroy(self, request, *args, **kwargs):
         logger.info(f"Cleanup: removed {n_removed} objects in total")
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+class ProjectJsonSchemaViewSet(CHORDPublicModelViewSet):
+    """
+    get:
+    Return list of ProjectJsonSchema
+
+    post:
+    Create a new ProjectJsonSchema
+    """
+
+    queryset = ProjectJsonSchema.objects.all().order_by("project__identifier")
```

Comment:
I think you can save a join here by just using `project_id`, which is the underlying name for the foreign key generated by the `project` django model